romeo/asset-contract/contracts/asset.clar

```diff
@@ -67,17 +75,19 @@
     (tree-depth uint)
     (block-header (buff 80)))
     (begin
-        (try! (is-contract-owner))
+        (asserts! (is-contract-owner) err-forbidden)
+        (asserts! (> amount u0) err-bad-request)
```

Comment:
The (ft-burn? ...) function used two lines below has a built-in check for a non-positive number (seen [here](https://docs.stacks.co/docs/clarity/language-functions#ft-burn)). 

Might confuse other developers to throw a custom error for a globally-known error. This line can/should be removed.

Reply:
Type is `uint` so it cannot be a negative value. Burning `u0` is also checked by the burn function itself so indeed this assertion is not necessary.